Javadoc fails to generate

Log:
```make

> Task :downloadVersionsManifest
:downloading minecraft versions manifest

> Task :downloadMcLibs
:downloading minecraft libraries

> Task :downloadIntermediary
Download https://github.com/Legacy-Fabric/Legacy-Intermediaries/raw/master/mappings/1.8.9.tiny

> Task :downloadMcJars
:downloading minecraft jars

> Task :mergeJars
:merging jars

> Task :mapIntermediaryJar
:mapping minecraft to intermediary

> Task :buildYarnTiny
:generating tiny mappings

> Task :invertIntermediary
:building inverted intermediary
Loading mapping file...
Rewriting mappings...
Done!

> Task :mergeTiny
:merging yarn and intermediary
Reading 1.8.9-intermediary-inverted.tiny
Reading yarn-mappings.tiny
Processing...
Done!
:reordering merged intermediary
Loading mapping file...
Rewriting mappings...
Done!

> Task :buildTinyWithEnum
:seeking auto-mappable fields
Found 1045 interesting names.

> Task :mapNamedJar
:mapping minecraft to named

> Task :genFakeSource FAILED

FAILURE: Build failed with an exception.

* Where:
Build file '/Volumes/FILES/curst/build.gradle' line: 690

* What went wrong:
Execution failed for task ':genFakeSource'.
> not a valid name: true

* Try:
Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output. Run with --scan to get full insights.

* Get more help at https://help.gradle.org

Deprecated Gradle features were used in this build, making it incompatible with Gradle 7.0.
Use '--warning-mode all' to show the individual deprecation warnings.
See https://docs.gradle.org/6.5/userguide/command_line_interface.html#sec:command_line_warnings

BUILD FAILED in 42s
13 actionable tasks: 13 executed

```

--- a/mappings/com/mojang/blaze3d/platform/GlHelper.mapping
+++ b/mappings/com/mojang/blaze3d/platform/GlHelper.mapping
@@ -1,0 +1,2 @@
+CLASS net/minecraft/class_1355 com/mojang/blaze3d/platform/GlHelper
+	METHOD method_4789 createContext ()V

--- a/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
+++ b/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
@@ -1,4 +1,5 @@
 CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
+	COMMENT Allows calling the static methods in {@code GL11} without actually touching it
 	FIELD field_4311 ALPHA_TEST Lnet/minecraft/class_1013$class_1014;
 		COMMENT Stores the alpha test state
 	FIELD field_4312 LIGHTING Lnet/minecraft/class_1013$class_1016;

--- a/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
+++ b/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
@@ -1,24 +1,44 @@
 CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	FIELD field_4311 ALPHA_TEST Lnet/minecraft/class_1013$class_1014;
+		COMMENT Stores the alpha test state
 	FIELD field_4312 LIGHTING Lnet/minecraft/class_1013$class_1016;
+		COMMENT Stores the lighting state
 	FIELD field_4313 LIGHTING_STATES [Lnet/minecraft/class_1013$class_1016;
+		COMMENT Stores all lighting states in an array
 	FIELD field_4314 COLOR_MATERIAL Lnet/minecraft/class_1013$class_1021;
+		COMMENT Stores the color material state
 	FIELD field_4315 BLEND Lnet/minecraft/class_1013$class_1015;
+		COMMENT Stores the blend state
 	FIELD field_4316 DEPTH Lnet/minecraft/class_1013$class_1023;
+		COMMENT Stores the depth testing state
 	FIELD field_4317 FOG Lnet/minecraft/class_1013$class_1024;
+		COMMENT Stores the fog state
 	FIELD field_4318 CULL Lnet/minecraft/class_1013$class_1022;
+		COMMENT Stores the cull state
 	FIELD field_4319 POLY_OFFSET Lnet/minecraft/class_1013$class_1025;
+		COMMENT Stores the polygon offset state
 	FIELD field_4320 COLOR_LOGIC Lnet/minecraft/class_1013$class_1019;
+		COMMENT Stores the color logic state
 	FIELD field_4321 TEX_GEN Lnet/minecraft/class_1013$class_1030;
+		COMMENT Stores the texture coordinate generation state
 	FIELD field_4322 CLEAR Lnet/minecraft/class_1013$class_1017;
+		COMMENT Stores the color clear state
 	FIELD field_4323 STENCIL Lnet/minecraft/class_1013$class_1027;
+		COMMENT Stores the stencil state
 	FIELD field_4324 NORMALIZE Lnet/minecraft/class_1013$class_1016;
+		COMMENT Stores the normalization state
 	FIELD field_4325 activeTexture I
+		COMMENT Stores the currently active texture
 	FIELD field_4326 TEXTURES [Lnet/minecraft/class_1013$class_1031;
+		COMMENT Stores all texture states in an array
 	FIELD field_4327 modelShadeMode I
+		COMMENT Stores the shading mode
 	FIELD field_4328 RESCALE_NORMAL Lnet/minecraft/class_1013$class_1016;
+		COMMENT Stores the normal rescale state
 	FIELD field_4329 COLOR_MASK Lnet/minecraft/class_1013$class_1020;
+		COMMENT Stores the color mask state
 	FIELD field_4330 COLOR Lnet/minecraft/class_1013$class_1018;
+		COMMENT Stores the color state in rgba
 	METHOD method_3393 disableNormalize ()V
 		COMMENT Disables Normalize
 	METHOD method_3394 enableRescaleNormal ()V
@@ -26,19 +46,25 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_3395 disableRescaleNormal ()V
 		COMMENT Disables rescale
 	METHOD method_3396 loadIdentity ()V
+		COMMENT Replaces the current matrix with the identity matrix
 	METHOD method_3397 pushMatrix ()V
+		COMMENT Pushes the current matrix stack
 	METHOD method_3398 popMatrix ()V
+		COMMENT Pops the current matrix stack
 	METHOD method_3399 clearColor ()V
+		COMMENT Clears the current color
 	METHOD method_3400 pushLightingAttributes ()V
 		COMMENT Pushes the attribute stack
 	METHOD method_3401 clearDepth (D)V
 		COMMENT Sets the clear value for the depth buffer
 		ARG 0 depth
 	METHOD method_3402 scaled (DDD)V
+		COMMENT Multiplies the current matrix by a general scaling matrix
 		ARG 0 x
 		ARG 2 y
 		ARG 4 z
 	METHOD method_3403 ortho (DDDDDD)V
+		COMMENT Multiplies the current matrix with an orthographic matrix
 		ARG 0 l
 		ARG 2 r
 		ARG 4 b
@@ -53,6 +79,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 0 factor
 		ARG 1 units
 	METHOD method_3406 scalef (FFF)V
+		COMMENT Multiplies the current matrix by a general scaling matrix
 		ARG 0 x
 		ARG 1 y
 		ARG 2 z
@@ -80,6 +107,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 2 srcFactorAlpha
 		ARG 3 dstFactorAlpha
 	METHOD method_3412 getFloat (ILjava/nio/FloatBuffer;)V
+		COMMENT Gets a float for a float buffer
 		ARG 0 mode
 		ARG 1 buffer
 	METHOD method_3413 enableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
@@ -95,6 +123,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 1 mode
 		ARG 2 buffer
 	METHOD method_3416 multiMatrix (Ljava/nio/FloatBuffer;)V
+		COMMENT Multiplies the current matrix with the specified matrix
 		ARG 0 buffer
 	METHOD method_3417 depthMask (Z)V
 		COMMENT Enables or disables writing into the depth buffer
@@ -108,6 +137,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_3419 popAttributes ()V
 		COMMENT Pops the attribute stack
 	METHOD method_3420 translated (DDD)V
+		COMMENT Multiplies the current matrix by a translation matrix
 		ARG 0 x
 		ARG 2 y
 		ARG 4 z
@@ -115,10 +145,12 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		COMMENT Sets fog parameters
 		ARG 0 start
 	METHOD method_3422 translatef (FFF)V
+		COMMENT Multiplies the current matrix by a translation matrix
 		ARG 0 x
 		ARG 1 y
 		ARG 2 z
 	METHOD method_3423 rotatef (FFFF)V
+		COMMENT Multiplies the current matrix by a rotation matrix
 		ARG 0 angle
 		ARG 1 x
 		ARG 2 y
@@ -145,10 +177,12 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		COMMENT Sets fog parameters
 		ARG 0 end
 	METHOD method_3430 color3f (FFF)V
+		COMMENT Sets the current color without alpha
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
 	METHOD method_3431 color4f (FFFF)V
+		COMMENT Sets the current color
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
@@ -206,10 +240,12 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_3452 disableFog ()V
 		COMMENT Disables fog
 	METHOD method_3453 matrixMode (I)V
+		COMMENT Allows specifying which matrix is the current matrix
 		ARG 0 mode
 	METHOD method_3454 enableCull ()V
 		COMMENT Enables culling
 	METHOD method_3455 callList (I)V
+		COMMENT Executes a display list
 		ARG 0 listId
 	METHOD method_3456 disableCull ()V
 		COMMENT Disables Culling

--- a/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
+++ b/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
@@ -1,5 +1,7 @@
 CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	FIELD field_4311 ALPHA_TEST Lnet/minecraft/class_1013$class_1014;
+	FIELD field_4312 LIGHTING Lnet/minecraft/class_1013$class_1016;
+	FIELD field_4313 LIGHTING_STATES [Lnet/minecraft/class_1013$class_1016;
 	FIELD field_4314 COLOR_MATERIAL Lnet/minecraft/class_1013$class_1021;
 	FIELD field_4315 BLEND Lnet/minecraft/class_1013$class_1015;
 	FIELD field_4316 DEPTH Lnet/minecraft/class_1013$class_1023;
@@ -10,17 +12,23 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	FIELD field_4321 TEX_GEN Lnet/minecraft/class_1013$class_1030;
 	FIELD field_4322 CLEAR Lnet/minecraft/class_1013$class_1017;
 	FIELD field_4323 STENCIL Lnet/minecraft/class_1013$class_1027;
+	FIELD field_4324 NORMALIZE Lnet/minecraft/class_1013$class_1016;
 	FIELD field_4325 activeTexture I
 	FIELD field_4326 TEXTURES [Lnet/minecraft/class_1013$class_1031;
 	FIELD field_4327 modelShadeMode I
+	FIELD field_4328 RESCALE_NORMAL Lnet/minecraft/class_1013$class_1016;
 	FIELD field_4329 COLOR_MASK Lnet/minecraft/class_1013$class_1020;
 	FIELD field_4330 COLOR Lnet/minecraft/class_1013$class_1018;
+	METHOD method_3393 disableNormalize ()V
 	METHOD method_3394 enableRescaleNormal ()V
+	METHOD method_3395 disableRescaleNormal ()V
 	METHOD method_3396 loadIdentity ()V
 	METHOD method_3397 pushMatrix ()V
 	METHOD method_3398 popMatrix ()V
 	METHOD method_3399 clearColor ()V
 	METHOD method_3400 pushLightingAttributes ()V
+	METHOD method_3401 clearDepth (D)V
+		ARG 0 depth
 	METHOD method_3402 scaled (DDD)V
 		ARG 0 x
 		ARG 2 y
@@ -46,6 +54,8 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 1 green
 		ARG 2 blue
 		ARG 3 alpha
+	METHOD method_3408 enableLight (I)V
+		ARG 0 index
 	METHOD method_3409 alphaFunc (IF)V
 		ARG 0 func
 		ARG 1 ref
@@ -58,7 +68,17 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 2 srcFactorAlpha
 		ARG 3 dstFactorAlpha
 	METHOD method_3412 getFloat (ILjava/nio/FloatBuffer;)V
+		ARG 0 mode
 		ARG 1 buffer
+	METHOD method_3413 enableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
+		ARG 0 coord
+	METHOD method_3414 genTex (Lnet/minecraft/class_1013$class_1028;I)V
+		ARG 0 coord
+		ARG 1 mode
+	METHOD method_3415 genTex (Lnet/minecraft/class_1013$class_1028;ILjava/nio/FloatBuffer;)V
+		ARG 0 coord
+		ARG 1 mode
+		ARG 2 buffer
 	METHOD method_3416 multiMatrix (Ljava/nio/FloatBuffer;)V
 		ARG 0 buffer
 	METHOD method_3417 depthMask (Z)V
@@ -84,9 +104,18 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 1 x
 		ARG 2 y
 		ARG 3 z
+	METHOD method_3424 disableLight (I)V
+		ARG 0 index
 	METHOD method_3425 blendFunc (II)V
 		ARG 0 srcFactor
 		ARG 1 dstFactor
+	METHOD method_3426 viewPort (IIII)V
+		ARG 0 x
+		ARG 1 y
+		ARG 2 width
+		ARG 3 height
+	METHOD method_3427 disableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
+		ARG 0 coord
 	METHOD method_3428 disableAlphaTest ()V
 	METHOD method_3429 fogEnd (F)V
 		ARG 0 end
@@ -106,6 +135,10 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_3434 enableAlphaTest ()V
 	METHOD method_3435 fogMode (I)V
 		ARG 0 mode
+	METHOD method_3436 enabledLighting ()V
+	METHOD method_3437 cullFace (I)V
+		ARG 0 mode
+	METHOD method_3438 disableLighting ()V
 	METHOD method_3439 logicOp (I)V
 		ARG 0 op
 	METHOD method_3440 enableColorMaterial ()V
@@ -124,6 +157,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_3449 enableBlend ()V
 	METHOD method_3450 enableFog ()V
 	METHOD method_3451 clear (I)V
+		ARG 0 mode
 	METHOD method_3452 disableFog ()V
 	METHOD method_3453 matrixMode (I)V
 		ARG 0 mode
@@ -138,6 +172,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	METHOD method_3461 enableTexture ()V
 	METHOD method_3462 disableTexture ()V
 	METHOD method_3463 getTexLevelParameter ()I
+	METHOD method_3464 enableNormalize ()V
 	CLASS class_1014 AlphaTestState
 		FIELD field_4332 capState Lnet/minecraft/class_1013$class_1016;
 		FIELD field_4333 func I
@@ -223,6 +258,10 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 			ARG 1 coord
 			ARG 2 id
 	CLASS class_1030 TexGenState
+		FIELD field_4388 s Lnet/minecraft/class_1013$class_1029;
+		FIELD field_4389 t Lnet/minecraft/class_1013$class_1029;
+		FIELD field_4390 r Lnet/minecraft/class_1013$class_1029;
+		FIELD field_4391 q Lnet/minecraft/class_1013$class_1029;
 	CLASS class_1031 Texture2DState
 		FIELD field_4392 capState Lnet/minecraft/class_1013$class_1016;
 		FIELD field_4393 boundTexture I

--- a/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
+++ b/mappings/com/mojang/blaze3d/platform/GlStateManager.mapping
@@ -20,14 +20,19 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 	FIELD field_4329 COLOR_MASK Lnet/minecraft/class_1013$class_1020;
 	FIELD field_4330 COLOR Lnet/minecraft/class_1013$class_1018;
 	METHOD method_3393 disableNormalize ()V
+		COMMENT Disables Normalize
 	METHOD method_3394 enableRescaleNormal ()V
+		COMMENT Enables rescale
 	METHOD method_3395 disableRescaleNormal ()V
+		COMMENT Disables rescale
 	METHOD method_3396 loadIdentity ()V
 	METHOD method_3397 pushMatrix ()V
 	METHOD method_3398 popMatrix ()V
 	METHOD method_3399 clearColor ()V
 	METHOD method_3400 pushLightingAttributes ()V
+		COMMENT Pushes the attribute stack
 	METHOD method_3401 clearDepth (D)V
+		COMMENT Sets the clear value for the depth buffer
 		ARG 0 depth
 	METHOD method_3402 scaled (DDD)V
 		ARG 0 x
@@ -41,8 +46,10 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 8 n
 		ARG 10 f
 	METHOD method_3404 fogDensity (F)V
+		COMMENT Sets fog parameters
 		ARG 0 density
 	METHOD method_3405 polygonOffset (FF)V
+		COMMENT Sets the scale and units used to calculate depth values
 		ARG 0 factor
 		ARG 1 units
 	METHOD method_3406 scalef (FFF)V
@@ -50,19 +57,24 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 1 y
 		ARG 2 z
 	METHOD method_3407 clearColor (FFFF)V
+		COMMENT Sets clear values for the color buffers
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
 		ARG 3 alpha
 	METHOD method_3408 enableLight (I)V
+		COMMENT Enables lighting for a certain element
 		ARG 0 index
 	METHOD method_3409 alphaFunc (IF)V
+		COMMENT Sets the alpha test function
 		ARG 0 func
 		ARG 1 ref
 	METHOD method_3410 colorMaterial (II)V
+		COMMENT Causes a material color to track the current color
 		ARG 0 face
 		ARG 1 mode
 	METHOD method_3411 blendFuncSeparate (IIII)V
+		COMMENT Sets pixel arithmetic for RGB and alpha components separately
 		ARG 0 srcFactorRGB
 		ARG 1 dstFactorRGB
 		ARG 2 srcFactorAlpha
@@ -71,29 +83,36 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 0 mode
 		ARG 1 buffer
 	METHOD method_3413 enableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
+		COMMENT Enables the generation of texture coordinates
 		ARG 0 coord
 	METHOD method_3414 genTex (Lnet/minecraft/class_1013$class_1028;I)V
+		COMMENT Allows controlling the generation of texture coordinates
 		ARG 0 coord
 		ARG 1 mode
 	METHOD method_3415 genTex (Lnet/minecraft/class_1013$class_1028;ILjava/nio/FloatBuffer;)V
+		COMMENT Allows controling the generation of texture coordinates with a float buffer
 		ARG 0 coord
 		ARG 1 mode
 		ARG 2 buffer
 	METHOD method_3416 multiMatrix (Ljava/nio/FloatBuffer;)V
 		ARG 0 buffer
 	METHOD method_3417 depthMask (Z)V
+		COMMENT Enables or disables writing into the depth buffer
 		ARG 0 mask
 	METHOD method_3418 colorMask (ZZZZ)V
+		COMMENT Enables or Disables writing of frame buffer color components
 		ARG 0 red
 		ARG 1 green
 		ARG 2 blue
 		ARG 3 alpha
 	METHOD method_3419 popAttributes ()V
+		COMMENT Pops the attribute stack
 	METHOD method_3420 translated (DDD)V
 		ARG 0 x
 		ARG 2 y
 		ARG 4 z
 	METHOD method_3421 fogStart (F)V
+		COMMENT Sets fog parameters
 		ARG 0 start
 	METHOD method_3422 translatef (FFF)V
 		ARG 0 x
@@ -105,19 +124,25 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 2 y
 		ARG 3 z
 	METHOD method_3424 disableLight (I)V
+		COMMENT Disables lighting for a certain element
 		ARG 0 index
 	METHOD method_3425 blendFunc (II)V
+		COMMENT Sets pixel arithmetic
 		ARG 0 srcFactor
 		ARG 1 dstFactor
 	METHOD method_3426 viewPort (IIII)V
+		COMMENT Sets the viewport
 		ARG 0 x
 		ARG 1 y
 		ARG 2 width
 		ARG 3 height
 	METHOD method_3427 disableTexCoord (Lnet/minecraft/class_1013$class_1028;)V
+		COMMENT Disables the generation of texture coordinates
 		ARG 0 coord
 	METHOD method_3428 disableAlphaTest ()V
+		COMMENT Disables alpha testing
 	METHOD method_3429 fogEnd (F)V
+		COMMENT Sets fog parameters
 		ARG 0 end
 	METHOD method_3430 color3f (FFF)V
 		ARG 0 red
@@ -129,50 +154,81 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		ARG 2 blue
 		ARG 3 alpha
 	METHOD method_3432 depthFunc (I)V
+		COMMENT Sets the value used for depth buffer comparisons
 		ARG 0 func
 	METHOD method_3433 getGenCoordState (Lnet/minecraft/class_1013$class_1028;)Lnet/minecraft/class_1013$class_1029;
+		COMMENT Retrieves texture coordinate generation state
 		ARG 0 coord
 	METHOD method_3434 enableAlphaTest ()V
+		COMMENT Enables alpha testing
 	METHOD method_3435 fogMode (I)V
+		COMMENT Sets fog parameters
 		ARG 0 mode
-	METHOD method_3436 enabledLighting ()V
+	METHOD method_3436 enableLighting ()V
+		COMMENT Enables lighting
 	METHOD method_3437 cullFace (I)V
+		COMMENT Sets whether front- or back-facing facets can be culled
 		ARG 0 mode
 	METHOD method_3438 disableLighting ()V
+		COMMENT Disables lighting
 	METHOD method_3439 logicOp (I)V
+		COMMENT Sets a logical pixel operation for color index rendering
 		ARG 0 op
 	METHOD method_3440 enableColorMaterial ()V
+		COMMENT Enables tracking of the current color by the material color
 	METHOD method_3441 activeTexture (I)V
+		COMMENT Sets the active texture unit
 		ARG 0 texture
 	METHOD method_3442 disableColorMaterial ()V
+		COMMENT Disables tracking of the current color by the material color
 	METHOD method_3443 deleteTexture (I)V
+		COMMENT Deletes a certain specified texture
 		ARG 0 texture
 	METHOD method_3444 disableDepthTest ()V
+		COMMENT Disables depth testing
 	METHOD method_3445 bindTexture (I)V
+		COMMENT Binds a certain specified texture to a texturing target
 		ARG 0 texture
 	METHOD method_3446 enableDepthTest ()V
+		COMMENT Enables depth testing
 	METHOD method_3447 shadeModel (I)V
+		COMMENT Sets shading mode as flat or smooth
 		ARG 0 mode
 	METHOD method_3448 disableBlend ()V
+		COMMENT Enables implementing transparency within objects
 	METHOD method_3449 enableBlend ()V
+		COMMENT Disables implementing transparency within objects
 	METHOD method_3450 enableFog ()V
+		COMMENT Enabled fog
 	METHOD method_3451 clear (I)V
+		COMMENT Clears buffers to preset values
 		ARG 0 mode
 	METHOD method_3452 disableFog ()V
+		COMMENT Disables fog
 	METHOD method_3453 matrixMode (I)V
 		ARG 0 mode
 	METHOD method_3454 enableCull ()V
+		COMMENT Enables culling
 	METHOD method_3455 callList (I)V
 		ARG 0 listId
 	METHOD method_3456 disableCull ()V
+		COMMENT Disables Culling
 	METHOD method_3457 enablePolyOffset ()V
+		COMMENT Enables Polygon Offset
 	METHOD method_3458 disablePolyOffset ()V
+		COMMENT Disables Polygon offset
 	METHOD method_3459 enableColorLogic ()V
+		COMMENT Enables color index rendering
 	METHOD method_3460 disableColorLogic ()V
+		COMMENT Disables color index rendering
 	METHOD method_3461 enableTexture ()V
+		COMMENT Enables active texture units
 	METHOD method_3462 disableTexture ()V
+		COMMENT Disables active texture units
 	METHOD method_3463 getTexLevelParameter ()I
+		COMMENT Generates texture names
 	METHOD method_3464 enableNormalize ()V
+		COMMENT Enables normalize
 	CLASS class_1014 AlphaTestState
 		FIELD field_4332 capState Lnet/minecraft/class_1013$class_1016;
 		FIELD field_4333 func I
@@ -251,6 +307,7 @@ CLASS net/minecraft/class_1013 com/mojang/blaze3d/platform/GlStateManager
 		FIELD field_4382 R Lnet/minecraft/class_1013$class_1028;
 		FIELD field_4383 Q Lnet/minecraft/class_1013$class_1028;
 	CLASS class_1029 TexGenCoordState
+		COMMENT
 		FIELD field_4385 capState Lnet/minecraft/class_1013$class_1016;
 		FIELD field_4386 coord I
 		FIELD field_4387 mode I

--- a/mappings/net/minecraft/block/Blocks.mapping
+++ b/mappings/net/minecraft/block/Blocks.mapping
@@ -38,7 +38,6 @@ CLASS net/minecraft/class_163 net/minecraft/block/Blocks
 	FIELD field_660 CACTUS Lnet/minecraft/class_169;
 	FIELD field_661 CLAY Lnet/minecraft/class_160;
 	FIELD field_662 AIR Lnet/minecraft/class_160;
-		COMMENT
 	FIELD field_663 SUGARCANE Lnet/minecraft/class_291;
 	FIELD field_664 JUKEBOX Lnet/minecraft/class_160;
 	FIELD field_665 OAK_FENCE Lnet/minecraft/class_160;
@@ -92,7 +91,6 @@ CLASS net/minecraft/class_163 net/minecraft/block/Blocks
 	FIELD field_713 LIT_REDSTONE_LAMP Lnet/minecraft/class_160;
 	FIELD field_714 DOBULE_WOODEN_SLAB Lnet/minecraft/class_226;
 	FIELD field_715 STONE Lnet/minecraft/class_160;
-		COMMENT
 	FIELD field_716 WOODEN_SLAB Lnet/minecraft/class_226;
 	FIELD field_717 COCOA Lnet/minecraft/class_160;
 	FIELD field_718 SANDSTONE_STAIRS Lnet/minecraft/class_160;

--- a/mappings/net/minecraft/class_1355.mapping
+++ b/mappings/net/minecraft/class_1355.mapping
@@ -1,2 +1,0 @@
-CLASS net/minecraft/class_1355
-	METHOD method_4789 createContext ()V

--- a/mappings/net/minecraft/world/level/LevelProperties.mapping
+++ b/mappings/net/minecraft/world/level/LevelProperties.mapping
@@ -102,7 +102,7 @@ CLASS net/minecraft/class_634 net/minecraft/world/level/LevelProperties
 	METHOD method_1951 setBorderCenterX (D)V
 		ARG 1 x
 	METHOD method_1953 setInitialized (Z)V
-		ARG 1 true
+		ARG 1 initialized
 	METHOD method_1954 getSpawnZ ()I
 	METHOD method_1955 setSafeZone (D)V
 		ARG 1 zone

